packages/components/src/styles/link-button-pure-styles.ts

```diff
@@ -76,6 +76,7 @@ export const getLinkButtonPureStyles = (
         ...buildResponsiveStyles(stretch, (responsiveStretch: boolean) => ({
           display: responsiveStretch ? 'block' : 'inline-block',
           width: responsiveStretch ? '100%' : 'auto', // prevents adjusting its size when used as flex or grid child
+          boxSizing: responsiveStretch ? 'border-box' : null,
```

Comment:
VRT coverage of this line? Border-Box changes spacing behaviour.

Reply:
Covered in Flyout-Navigation. But we can also add a dedicated test for Link-Pure and Button-Pure VRT test page

Reply:
I solved it without changing the behaviour of `link-button-pure-styles.ts`